solutions/src/main/java/ru/sberbank/school/task07/FileParserImpl.java

```diff
@@ -0,0 +1,27 @@
+package ru.sberbank.school.task07;
+
+import java.io.*;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+import java.util.Scanner;
+
+/**
+ * Created by Mart
+ * 01.06.2019
+ **/
+public class FileParserImpl implements FileParser {
+    @Override
+    public List<String> parse(String pathToFile) throws FileNotFoundException {
+        Objects.requireNonNull(pathToFile, "Path to file is null");
+
+        List<String> list = new ArrayList<>();
+        File file = new File(pathToFile);
+        Scanner scanner = new Scanner(file, "UTF-8");
+
+        while (scanner.hasNext()) {
+             list.add(scanner.nextLine().toLowerCase());
```

Comment:
&#39;while&#39; child has incorrect indentation level 13, expected level should be 12.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a-1727303112>*